src/vm/customattribute.cpp

```diff
@@ -294,13 +294,11 @@ HRESULT Attribute::ParseCaValue(
         IfFailGo(ca.GetU4(&len));
         pCaArg->arr.length = len;
         pCaArg->arr.pSArray = NULL;
-        if (pCaArg->arr.length == (ULONG)-1)
-            break;
 
         IfNullGo(pCaArg->arr.pSArray = pCaValueArrayFactory->Create()); 
```

Comment:
Received your message, again give me some minutes to double check this and add a new commit plus a reply/comment